pkg/util/utils.go

```diff
@@ -109,3 +110,22 @@ func MaybeChownDirRecursiveToMinikubeUser(dir string) error {
 func ParseKubernetesVersion(version string) (semver.Version, error) {
 	return semver.Make(version[1:])
 }
+
+// NormalizeImageName automatically tag latest to image
+// Example:
+//  nginx -> nginx:latest
+//  localhost:5000/nginx -> localhost:5000/nginx:latest
+//  localhost:5000/nginx:latest -> localhost:5000/nginx:latest
+//  docker.io/dotnet/core/sdk -> docker.io/dotnet/core/sdk:latest
+func NormalizeImageName(image string) string {
```

Comment:
no need to have this in util package. I think `image` package is best place for it.
and it could be a private func in that package.

it could be called normalizeTagName

Reply:
done